discovery-provider/src/tasks/index.py

```diff
@@ -238,14 +240,17 @@ def fetch_ipfs_metadata(
                 user_contract.events, user_event_types_lookup["update_multihash"]
             )().processReceipt(tx_receipt)
             for entry in user_events_tx:
+                event_args = entry["args"]
                 metadata_multihash = helpers.multihash_digest_to_cid(
```

Comment:
```suggestion
                cid = helpers.multihash_digest_to_cid(
```
nit but what do you think of doing this?

Reply:
done